cli/cmd/cmd.go

```diff
@@ -206,6 +206,8 @@ func (c *ServicedCli) cmdInit(ctx *cli.Context) error {
 		fmt.Fprintf(os.Stderr, "Unable to set logging options: %s\n", err)
 	}
 
+	isvcs.InitAllIsvcs()
```

Comment:
Ok I made the move your requested for this but it causes an unrecoverable error on start on both CLI and master service operations.
```
Unable to set isvcs options: Error setting internal service environment:'zookeeper:ZKID=1'  Service 'zookeeper' must be one of []'
Error setting internal service environment:'zookeeper:ZKID=1'  Service 'zookeeper' must be one of []'
```
Lets discuss.